### DISCUSSION TITLE
Questions about your "Emergence" graph timelapse video

### DISCUSSION THREAD

**Discussion by Oshyan** [BryanJenksCommunity/FAQ#41] [Q&A] [answered]
I really enjoyed seeing your "Emergence" graph timelapse: https://youtu.be/8Xvbp0LIFqs

It's really cool looking, but without knowing the contents and history of your knowledgebase it feels impossible to derive any meaning from it as an observer. I wonder more whether you have a sense, or even a precise idea, of what is happening at any given moment, or at least at major transitions. In other words does it have actual value and additional information for you as the writer, or is it also mostly just pretty?

I'm also really curious what those long, linearly linked structures are that start appearing a little after 1m.

**Comment by tallguyjenks** [comment, marked as answer]
> ... It feels impossible to derive any meaning from it as an observer. 

Yeah it probably is!

> I wonder more whether you have a sense, or even a precise idea, of what is happening at any given moment, or at least at major transitions. 

I'm not sure if the timelapse is the chronological order of files creation or not. and I've done several operations on my vault over time that make that an inaccurate gage to observe. I can easily identify visual structures, but points in time of the timelapse not so much.

> In other words does it have actual value and additional information for you as the writer, or is it also mostly just pretty?

The global graph does have value for a particular reason, the way I see my emerging MOC's through my `[[tags]]` beyond that, it's just pretty and the local graph is more useful

> I'm also really curious what those long, linearly linked structures are that start appearing a little after 1m.

the long linked red nodes are my daily notes linked via breadcrumbs like this:

`[[2021-03-10]] <== <button class="date_button_today">Today</button> ==> [[2021-03-12]]`

**Reply by Oshyan** [reply]
Excellent, thank you! That is all roughly as I imagined, then. 

> I'm not sure if the timelapse is the chronological order of files creation or not.

Yes, did they not provide any info with release of this feature as to how it works?

**Reply by tallguyjenks** [reply]
They may have, but I didn't see it explicitly. I've messed with my vault so frequently with scripting languages and other tools that at this point the timelapse is little more than a cool movie to watch and doesn't really hold any particular meaning or academic value to me :)
